server/graphql/v2/object/Host.js

```diff
@@ -1232,12 +1232,34 @@ export const GraphQLHost = new GraphQLObjectType({
           }
 
           if (args.isUnhosted) {
-            const unhostedActivities = await models.Activity.findAll({
-              where: { type: 'collective.unhosted', HostCollectiveId: host.id },
+            const formerHostedMembers = await models.Member.findAll({
+              where: { MemberCollectiveId: host.id, role: 'HOST', deletedAt: { [Op.ne]: null } },
```

Comment:
I wouldn't trust that: a collective could be unhosted, then rejoin the host.

Since the `HostApplication` is required, I don't think we need to look at members at all. And I don't think we need to do it in two queries either. I would suggest replacing this query by:
```ts
where.HostCollectiveId = { [Op.or]: [{ [Op.ne]: host.id }, { [Op.is]: null }] };
include.push([{
  association: 'hostApplications',
  attributes: [],
  required: true,
  where: { HostCollectiveId: host.id, status: 'APPROVED' }
}])
```

Then remove the `where.id`, and add the include on the main query.

Reply:
LOL, checkout the query this generates:
```sql
SELECT
  "Collective".*
FROM (
  SELECT
    "Collective"."id",
    "Collective"."type",
    "Collective"."slug",
    "Collective"."name",
    "Collective"."legalName",
    "Collective"."company",
    "Collective"."CreatedByUserId",
    "Collective"."LastEditedByUserId",
    "Collective"."ParentCollectiveId",
    "Collective"."HostCollectiveId",
    "Collective"."hostFeePercent",
    "Collective"."platformFeePercent",
    "Collective"."description",
    "Collective"."longDescription",
    "Collective"."expensePolicy",
    "Collective"."contributionPolicy",
    "Collective"."currency",
    "Collective"."image",
    "Collective"."backgroundImage",
    "Collective"."countryISO",
    "Collective"."settings",
    "Collective"."data",
    "Collective"."startsAt",
    "Collective"."endsAt",
    "Collective"."timezone",
    "Collective"."createdAt",
    "Collective"."updatedAt",
    "Collective"."isActive",
    "Collective"."isIncognito",
    "Collective"."approvedAt",
    "Collective"."twitterHandle",
    "Collective"."repositoryUrl",
    "Collective"."website",
    "Collective"."tags",
    "Collective"."deactivatedAt",
    "Collective"."isHostAccount",
    "Collective"."plan",
    "Collective"."deletedAt"
  FROM
    "Collectives" AS "Collective"
  WHERE ("Collective"."deletedAt" IS NULL
    AND (("Collective"."HostCollectiveId" != 11004
        OR "Collective"."HostCollectiveId" IS NULL)
      AND "Collective"."id" != 11004
      AND "Collective"."type" IN ('COLLECTIVE', 'FUND')))
  AND (
    SELECT
      "CollectiveId"
    FROM
      "HostApplications" AS "hostApplications"
    WHERE (("hostApplications"."deletedAt" IS NULL
        AND ("hostApplications"."HostCollectiveId" = 11004
          AND "hostApplications"."status" = 'APPROVED'))
      AND "hostApplications"."CollectiveId" = "Collective"."id")
  LIMIT 1) IS NOT NULL
ORDER BY
  "Collective"."approvedAt" DESC
LIMIT 20 OFFSET 0) AS "Collective"
  INNER JOIN "HostApplications" AS "hostApplications" ON "Collective"."id" = "hostApplications"."CollectiveId"
    AND ("hostApplications"."deletedAt" IS NULL
      AND ("hostApplications"."HostCollectiveId" = 11004
        AND "hostApplications"."status" = 'APPROVED'))
ORDER BY
  "Collective"."approvedAt" DESC;
```